config/settings.yml

```diff
@@ -27,6 +27,11 @@ github_notification_monitor:
   repo_names: []
 merge_target_titler:
   enabled_repos: []
+path_based_labeler:
+  enabled_repos: []
+  rules: []
+    # - pattern:
+    #   label:
```

Comment:
enabled_repos is the old enable/disable functionality that I'm trying to revamp and make more common.  For now it has to stay though.

Even though it's kind of clunky, perhaps

```yaml
path_based_labeler:
  enabled_repos:
    ManageIQ/manageiq:
    - pattern:
       label:
    - pattern:
       label: